tests/perforce_utils.py

```diff
@@ -153,6 +160,12 @@ def docker_perforce(request):
                     pass
 
 
+class PerforceConnection:
```

Comment:
The "Any" is still used in the __init__ method of the PerforceWorkspace class. This should be fixed.